### PR TITLE
[Docs] revDatabaseColumnNumbered and revOpenDatabase parameters

### PR DESCRIPTION
**Pull request by seaniepie**
- All parameters camelCased
- New overall syntax added to revOpenDatabase to include databaseType
- Parameter types added
- Parameter formatting improved

### CODE REVIEW

File: docs/dictionary/function/revOpenDatabase.lcdoc [hunk 1]
```diff
@@ -50,12 +51,12 @@ Example:
 get revOpenDatabase("postgresql", "192.168.1.100", "dbName", myUsr, myPass, "sslmode=require", "sslcompression=0")
 
 Parameters:
-host (string):
+dbHost (string):
```
Comment (montegoulding) [inline docs/dictionary/function/revOpenDatabase.lcdoc]: @seaniepie I think `host:port` should be combined into a single `address` parameter description.

**Comment by peter-b** [comment]
@livecode-vulcan review ok d64f40e

**Comment by livecode-vulcan** [comment]
💙 review by @peter-b ok d64f40eb9a4e94ab6cf23e60fab4ce69d3894739

**Comment by livecode-vulcan** [comment]
😎 test success d64f40eb9a4e94ab6cf23e60fab4ce69d3894739

* try-community-linux-x86: [success](https://vulcan.livecode.com/builders/try-community-linux-x86/builds/647)
* try-community-mac: [success](https://vulcan.livecode.com/builders/try-community-mac/builds/652)
* try-community-linux-x86_64: [success](https://vulcan.livecode.com/builders/try-community-linux-x86_64/builds/647)
* try-community-emscripten: [success](https://vulcan.livecode.com/builders/try-community-emscripten/builds/651)
* try-community-ios: [success](https://vulcan.livecode.com/builders/try-community-ios/builds/656)
* try-community-win-x86: [success](https://vulcan.livecode.com/builders/try-community-win-x86/builds/648)
* try-community-android-armv6: [success](https://vulcan.livecode.com/builders/try-community-android-armv6/builds/648)

**Pull request merged**